frontend/src/Metamaps/PasteInput.js

```diff
@@ -70,11 +67,23 @@ const PasteInput = {
       Import.handleURL(text, coords)
     } else if (text[0] === '{') {
       Import.handleJSON(text)
-    } else if (text.match(/\t/)) {
+    } else if (text.match(/[Tt]opics\t/) || text.match(/[Ss]ynapses\t/)) {
```

Comment:
I'd change these regexes to match the start of the string. I'd have to look it up, but I think it's `^` to match the start of a string